src/briefcase/platforms/iOS/__init__.py

```diff
@@ -14,7 +14,7 @@ def verify_tools(self):
         # Require XCode 10.0.0. There's no particular reason for this
         # specific version, other than it's a nice round number that's
         # not *that* old at time of writing.
-        verify_xcode_install(self.tools, min_version=(10, 0, 0))
+        Xcode.verify(self.tools, min_version=(10, 0, 0))
```

Comment:
This is no longer correct - we definitely need at least Xcode 13.0 at this point, due to file formats.

Reply:
bumped to 13.0.0